src/main/java/nova/core/recipes/crafting/CraftingGrid.java

```diff
@@ -16,7 +16,7 @@
 	String typeCrafting = "crafting";
 
 	/**
-	 * Represents the player that is currently using this crafting grid. Can be null.
+	 * Represents the player that is currently using this crafting grid. Can be an {@code Optional.empty()}.
```

Comment:
If it's optional at all it can be an Optional.empty, if it couldn't, then it wouldn't be an optional...